tests/phpunit/tests/blocks/editor.php

```diff
@@ -454,6 +454,14 @@ public function test_wp_get_post_content_block_attributes() {
 		$this->assertSame( $attributes_with_layout, wp_get_post_content_block_attributes() );
 	}
 
+	public function test_wp_get_post_content_block_attributes_no_layout() {
+		$attributes_empty = array();
+
+		switch_theme( 'block-theme-post-content-default' );
+
+		$this->assertSame( $attributes_empty, wp_get_post_content_block_attributes() );
```

Comment:
Nit-pick: No need for a variable here, particularly as it's something that simple.
```suggestion
		switch_theme( 'block-theme-post-content-default' );

		$this->assertSame( array(), wp_get_post_content_block_attributes() );
```